aws/logs_monitoring/installation_alternatives.md

```diff
@@ -33,6 +33,36 @@ resource "aws_cloudformation_stack" "datadog-forwarder" {
   template_url = "https://datadog-cloudformation-template.s3.amazonaws.com/aws/forwarder/latest.yaml"
   
 }
+
+# Example Cloudwatch Logs subscription for AWS Batch
+
+variable "aws_region" {
+  type        = string
+  description = "AWS Region"
+  default     = "us-east-1"
+}
+
+variable "account_id" {
+  type = string
+  description = "AWS Account Id"
+}
+
+resource "aws_cloudwatch_log_subscription_filter" "aws_batch" {
+  name                  = "aws_batch"
+  log_group_name        = "/aws/batch/job"
+  destination_arn       = aws_cloudformation_stack.datadog-forwarder.outputs["DatadogForwarderArn"]
+  filter_pattern        = ""
+}
+
+resource "aws_lambda_permission" "datadog_forwarder" {
```

Comment:
This is not needed, the forwarder CFN template already allows invocation from any log group. This is because Lambda actually have limit on resource-based policy blob size, enumerating permissions group by group won't work for large customers.

Reply:
Makes sense thanks